packages/react-form/src/hooks/list/test/dynamiclist.test.tsx

```diff
@@ -39,6 +57,36 @@ describe('useDynamicList', () => {
       );
     }
 
+    it('can move field to a new position', () => {
```

Comment:
Would it be possible to separate this test into move up and move down?

so a describe('can move field to a new position') which will then have two tests (moving up and moving down) ?

Reply:
yes, that's a good suggestion.

Reply:
But do we really need to test the up/down case? The way this is implemented, it's completely agnostic of what the move is. I personally wouldn't clutter tests with redundant cases.

Reply:
i.e. I would just test one move and be done with it.